ruby/main.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'app'
+
+def main
+  print "\nWelcome to catalog of my things\n"
+  app = App.new
+  app.start
+  loop do
+    print_options
+    option = gets.chomp.to_i
+
+    handle_case(option, app)
+
+    break if option == 13
+  end
+end
+
+def handle_case(option, app)
+  case option
+  when 1
+    app.list_books
+  when 2
+    print app.list_music_albums
+  when 3
+    print app.list_movies
+  when 4
+    app.list_games
+  when 5
+    print app.list_genres
+  when 6
+    app.list_labels
+  when 7
+    app.list_authors
+  when 8
+    app.list_sources
+  when 9
+    app.add_book
+  when 10
+    print app.add_music_album
+  when 11
+    print app.add_movie
+  when 12
+    app.add_game
+  when 13
+    app.leave
+    print 'Thanks for using the app!'
```

Comment:
- [optional] kindly check the exit message, as it does not print out when I exit the app. Great job.